tests/gradients/core/test_pulse_gradient.py

```diff
@@ -1327,7 +1327,7 @@ def circuit(params):
     @pytest.mark.skip("Applying this gradient transform to QNodes directly is not supported.")
     def test_qnode_expval_single_par(self, dev_name):
         """Test that a simple qnode that returns an expectation value
-        can be differentiated with pulse_generator."""
+        can be differentiated with pulse_odegen."""
```

Comment:
Maybe I'm misunderstanding something here but this test seems to use ``stoch_pulse_grad``, isn't that the other differentiation method?